server/channels/store/sqlstore/team_store.go

```diff
@@ -1459,6 +1459,29 @@ func (s SqlTeamStore) GetAllForExportAfter(limit int, afterId string) ([]*model.
 	return data, nil
 }
 
+// GetForExport returns a team for export, where Teams.Id is equal to the teamID passed as parameter.
+func (s SqlTeamStore) GetForExport(teamID string) (*model.TeamForExport, error) {
+	data := model.TeamForExport{}
+	query, args, err := s.getQueryBuilder().
+		Select("Teams.*", "Schemes.Name as SchemeName").
```

Comment:
Why do we join with Schemes table here?

Reply:
Because we return a `TeamForExport`, which has the same fields as a `Team` plus a `SchemeName`, taken from the `Schemes` table. (The `GetForExport` function just imitates `GetAllForExport` which does the same join.)

Reply:
Ah okay, missed that.